src/EntityFramework/Core/Mapping/MappingItemLoader.cs

```diff
@@ -2092,7 +2090,7 @@ private void ValidateFunctionImportMappingParameters(XPathNavigator nav, EdmFunc
             var lineInfo = new LineInfo(nav);
 
             // Process entity type.
-            var entityTypeString = GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
+            GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
```

Comment:
Yes - we're just doing this to get the code working. And I thought we had agreed to do the simplest thing to get it working. So generally I have added suppressions - but in some cases it seemed that there was no reason for the code (or maybe the reason was previous refactoring which had failed to remove some unused code). Unfortunately, I'm not in today and email is flaky. What I'd like to do is do the minimum to get it working and get this going on the Dev15 CI and then revisit the whole CodeAnalysis question (whether we should do it at all and what value it adds / costs) later. @divega has asked me to raise a bug for that. There are only a small number of these cases where I wasn't sure about side-effects (which I tried to mark with the `//LAJ comment` though I missed this one). I thought we could just manually review those few. But if we don't have any experts who can definitely say that the code has no side-effects then I can just suppress the message and we can revisit as part of the bigger issue later?